### PR TITLE
Remove legacy repository from capabilities / ssl / form fields

### PR DESCRIPTION
**Pull request by adreghiciu**
https://issues.sonatype.org/browse/NEXUS-8505
http://bamboo.s/browse/NX3-OSSF243

* ported capabilities & dropped one condition: repository is not blocked condition (unused, we will make an equivalent when needed)
* ported repository combobox (backend to actually do the filtering is not done)
* removed ssl repository capability as ssl is handled differently in cma repos, without a capability

### CODE REVIEW

**Comment by adreghiciu** [comment]
CI :+1: http://bamboo.s/browse/NX3-OSSF243-1

File: plugins/capabilities/nexus-capabilities-plugin/src/main/java/org/sonatype/nexus/capability/condition/RepositoryOnlineCondition.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  }
+
+  @Override
+  public String toString() {
+    try {
+      final String repositoryName = getRepositoryName();
+      return String.format("Repository '%s' is online", repositoryName);
+    }
+    catch (Exception ignore) {
+      return String.format("Repository '(could not be evaluated)' is online");
```
Comment (mrprescott) [inline plugins/capabilities/nexus-capabilities-plugin/src/main/java/org/sonatype/nexus/capability/condition/RepositoryOnlineCondition.java]: Probably don't need String.format() for this.

File: plugins/capabilities/nexus-capabilities-plugin/src/main/java/org/sonatype/nexus/capability/condition/RepositoryOnlineCondition.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  }
+
+  @Override
+  public String explainSatisfied() {
+    try {
+      final String repositoryName = getRepositoryName();
+      return String.format("Repository '%s' is online", repositoryName);
+    }
+    catch (Exception ignore) {
+      return String.format("Repository '(could not be evaluated)' is online");
```
Comment (mrprescott) [inline plugins/capabilities/nexus-capabilities-plugin/src/main/java/org/sonatype/nexus/capability/condition/RepositoryOnlineCondition.java]: String.format() unnecessary, and again on line 95.

**Comment by mrprescott** [comment]
+1 w/ minor comments

**Comment by jdillon** [comment]
+1

**Pull request merged**